analysis/deep2_r23_o32/general.py

```diff
@@ -386,19 +394,10 @@ def run_individual_functions(fitspath, want, dataset='n_Bins', n_split=3, adapti
             error_prop.fluxes_derived_prop(fitspath, raw=True, binned_data=True, apply_dust=True, revised=True)
 
         # Run Monte Carlo randomization calculations (option to apply dust correction)
-        error_prop.fluxes_derived_prop(fitspath, raw=False, binned_data=True, apply_dust=False, revised=False)
+        error_prop.fluxes_derived_prop(fitspath, raw=False, binned_datha=True, apply_dust=False, revised=False)
```

Comment:
Um, this seems to be a new typo that was introduced. Are you sure it ran completely fine?

Reply:
Fixed.